examples/zephyr/rpc/src/main.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * copyright (c) 2023 golioth, inc.
+ *
+ * spdx-license-identifier: apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(rpc_sample, LOG_LEVEL_DBG);
+
+#include "golioth.h"
+#include <samples/common/hardcoded_credentials.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+
+#include <samples/common/net_connect.h>
+
+golioth_client_t client;
```

Comment:
That is not needed, right? Could you define it in `main()`?

Reply:
Yes, I moved that as you recommended.

I also put all the includes together, without the space, in alphabetical order.